src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,24 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence using the Fisher-Yates shuffle algorithm.
```

Comment:
I think it's a fair point, if it's in the public doc changing it is a breaking change of sorts. The algorithm is more of an implementation note, happy to have links there.